src/App.scss

```diff
@@ -0,0 +1,286 @@
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 10px;
+}
+
+&__self-inc-label {
+    width: 25%;
+    display: block;
+    padding-left: 31%;
+    font-size: larger;
+    margin-right: 13px;
+}
+
+  &__self-inc-field {
+    box-sizing: border-box;
+    border: 1px black solid;
+    border-radius: 5px;
+    height: 25px;
+    width: 27%;
+    font-size: large;
+    background-color: lightyellow;
+    margin-right: 30%;
+}
+
+&__form-self-credit {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 10px;
+}
+
+&__self-credit-label {
+    width: 25%;
+    display: block;
+    padding-left: 27%;
+    font-size: larger;
+    margin-right: 13px;
+}
+
+&__self-credit-field {
+    box-sizing: border-box;
+    border: 1px black solid;
+    border-radius: 5px;
+    height: 25px;
+    width: 31%;
+    font-size: large;
+    background-color: lightyellow;
+    margin-right: 22%;
+}
+
+&__form-part-income {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 10px;
+}
+
+&__part-inc-label {
+    width: 29%;
+    display: block;
+    padding-left: 28%;
+    font-size: larger;
+    margin-right: 13px;
+}
+
+&__part-inc-field {
+    box-sizing: border-box;
+    border: 1px black solid;
+    border-radius: 5px;
+    height: 25px;
+    width: 27%;
+    font-size: large;
+    background-color: lightyellow;
+    margin-right: 30%;
+}
+
+&__form-part-credit {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 10px;
+}
+
+&__part-credit-label {
+    width: 36%;
+    display: block;
+    padding-left: 21.5%;
+    font-size: larger;
+    margin-right: 13px;
+}
+
+&__part-credit-field {
+    box-sizing: border-box;
+    border: 1px black solid;
+    border-radius: 5px;
+    height: 25px;
+    width: 36%;
+    font-size: large;
+    background-color: lightyellow;
+    margin-right: 22%;
+}
+
+&__form-security {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 10px;
+}
+
+&__security-label {
+    width: 23%;
+    display: block;
+    padding-left: 38.5%;
+    font-size: larger;
+    margin-right: 13px;
+}
+
+&__form-security-option {
+    box-sizing: border-box;
+    border: 1px black solid;
+    border-radius: 5px;
+    height: 25px;
+    width: 61%;
+    font-size: large;
+    background-color: lightyellow;
+    margin-right: 22%;
+}
+
+
+&__form-selfEmp {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 10px;
+}
+
+&__selfEmp-ind-label {
+    width: 60%;
+    display: block;
+    padding-left: 22%;
+    font-size: larger;
+    margin-right: 13px;
+}
+
+&__selfEmp-ind-field {
+    box-sizing: border-box;
+    border: 1px black solid;
+    border-radius: 5px;
+    height: 25px;
+    width: 61%;
+    font-size: large;
+    background-color: lightyellow;
+    margin-right: 22%;
+}
+
+&__button {
+    width: 250px;
+    margin-left: 40%;
+    background-color: #379c37;
+    color: white;
+    font-size: 24px;
+    height: 3rem;
+    border-radius: 10px;
+    margin-top: 15px;
+}
+}
```

Comment:
Prefer semantic classes rather than 'pointing' classes like these. So instead of saying 'this button is blue', say 'whatever has this class attached is blue'

Reply:
I agree with Monarch, there are a lot of repeated styles heres. Mixins would be helpful as well to decrease typing since you're using Sass. Look up utility classes too.

Reply:
Excellent feedback ... thanks.

Reply:
You should see what it looks like when I get responsive design in play .... lol